templates/base/next.config.js

```diff
@@ -1,13 +1,30 @@
+require('dotenv').config();
```

Comment:
Lets avoid using this package as there are many ways we can access env variables and .env files is something we are going to discourage as they sometimes lead to security issues.

Reply:
sure will remove it.